enterprise/frontend/src/metabase-enterprise/embedding_iframe_sdk/embed.ts

```diff
@@ -383,8 +383,6 @@ class MetabaseEmbed {
           error,
         });
       }
-
-      throw error;
```

Comment:
We now use the custom elements syntax, so it no longer makes sense to throw errors to users, because they can't catch it anymore.